api/proto/flowlog/flowlog.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2017-2019 Zededa, Inc.
+// All rights reserved.
```

Comment:
Please fix to have new copyright/license comment as in the existing files.

Reply:
Thanks, fixed it.